postal/internal/dependency_mirror_test.go

```diff
@@ -213,14 +213,34 @@ func testDependencyMirror(t *testing.T, context spec.G, it spec.S) {
 			})
 
 			it("defaults to environment variable and ignores binding", func() {
-				boundDependency, err := resolver.FindDependencyMirror("some-uri", "some-platform-dir")
+				boundDependency, err := resolver.FindDependencyMirror("https://some-uri.com/dep.tgz", "some-platform-dir")
 				Expect(err).ToNot(HaveOccurred())
-				Expect(boundDependency).NotTo(Equal("https://mirror.example.org/some-uri"))
-				Expect(boundDependency).To(Equal("https://mirror.other-example.org/some-uri"))
+				Expect(boundDependency).NotTo(Equal("https://mirror.example.org/some-uri.com/dep.tgz"))
+				Expect(boundDependency).To(Equal("https://mirror.other-example.org/some-uri.com/dep.tgz"))
 
 			})
 		})
 
+		context("without originalHost placeholder", func() {
```

Comment:
what does this mean?

Reply:
Can you elaborate on what the bug that you're solving is?

Reply:
The user can use `{originalHost}` as a placeholder to include the original url path when specifying a mirror. For example, if the original url is `https://artifacts.paketo.io/nginx_1.24.0_linux_x64_jammy_fa1dfe46.tgz` and the mirror is `https://user:pass@local-mirror.example.com/buildpacks-dependencies/{originalHost}`, then this will translate to `https://user:pass@local-mirror.example.com/buildpacks-dependencies/artifacts.paketo.io/nginx_1.24.0_linux_x64_jammy_fa1dfe46.tgz`. But if the placeholder is excluded, then the mirror translates to `https://user:pass@local-mirror.example.com/buildpacks-dependencies/nginx_1.24.0_linux_x64_jammy_fa1dfe46.tgz`